front-end/src/App.tsx

```diff
@@ -50,9 +50,9 @@ function App() {
 
       // Keep track of a Node representing our own server
       setNodes([...nodes, {
-        "host": host,
-        "username": loggedInUser?.username, 
-        "password": loggedInUser?.password
+        host: host,
+        username: loggedInUser?.username, 
+        password: loggedInUser?.password
```

Comment:
Just fixing up some old code - was too used to Python's definition of dictionaries and didn't realize you could define keys without quotes.